test/hpke_test.c

```diff
@@ -1008,11 +1008,10 @@ static int test_hpke_modes_suites(void)
                             overallresult = 0;
                     }
                     if (COIN_IS_HEADS) {
-                        RAND_bytes_ex(testctx,
-                                      (unsigned char *) &startseq,
-                                      sizeof(startseq),
-                                      RAND_DRBG_STRENGTH);
-                        if (!TEST_true(OSSL_HPKE_CTX_set_seq(ctx, startseq)))
+                        if (!TEST_int_eq(1, RAND_bytes_ex(testctx,
```

Comment:
TEST_false or, perhaps better, TEST_int_le ?

Reply:
It is documented as returning 1, -1 or 0. I think this is sufficient.

Reply:
I'd like the -1 check at least.  Continuing after an error isn't good style in a test case....

Reply:
I think you are making a comment on the entire test...
i.e. every single call works this way for this test.

Reply:
If this line changes then the whole test should change.